### DISCUSSION TITLE
Is it possible to change the shortcut for the Helm transient state?

### DISCUSSION THREAD

**Discussion by geoffreyvanwyk** [syl20bnr/spacemacs#16233] [Q&A] [answered]
The default shortcut for the Helm transient state is `M-SPC`, but on Windows, that key combination is irrevocably bound to the shortcut menu of the active window.

Is it possible to change the shortcut for the Helm transient state instead?

**Comment by alexey0308** [comment]
hi @geoffreyvanwyk, there is yet another binding `s-M-SPC` - does it fit your needs?

**Reply by geoffreyvanwyk** [reply]
Hi @alexey0308 , does the small `s` mean `SHIFT`? I tried with `SHIFT-M-SPC` but failed. It is not used by Windows 10 ([Windows Keyboard Shortcuts](https://support.microsoft.com/en-us/windows/keyboard-shortcuts-in-windows-dcc61a57-8ff0-cffe-9796-cb9706c75eec#WindowsVersion=Windows_10)).

With an actual s in `s-M-SPC`, it interprets it as `M-s M-s` which is undefined or shows isearch.

**Reply by alexey0308** [reply, marked as answer]
yes, I meant `shift`. On linux it works though, i.e. when I start any helm command like `M-x` and then do shift-alt-space, I can use the transient commands, but I must confess I did not know about this functionality before this question.

You can take a look how it is set [in the layer config](https://github.com/syl20bnr/spacemacs/blob/develop/layers/%2Bspacemacs/spacemacs-completion/packages.el#L120) and try to set your own keybinding in the `user-config`.

**Reply by geoffreyvanwyk** [reply]
Thanks @alexey0308 .

I added the following to `user-config` and it is working now:

```elisp
  (with-eval-after-load 'helm
    (define-key helm-map (kbd "M-RET")
                'spacemacs/helm-navigation-transient-state/body))
```
